src/workflows/rna/rna_singlesample/test.nf

```diff
@@ -19,7 +19,8 @@ workflow test_wf {
         max_fraction_mito: 0.2,
         var_gene_names: "gene_symbol",
         var_name_mitochondrial_genes: "mitochondrial",
-        obs_name_mitochondrial_fraction: "fraction_mitochondrial",
+        // see if obs_name is automatically filled in
```

Comment:
Is there still a test case for when the user does specify the fraction?

Reply:
Yes -- there is a second test